assets/js/admin/admin-scripts.js

```diff
@@ -721,7 +672,63 @@ var give_setting_edit = false;
 					});
 				}
 			}).change();
+		},
+
+		saveButtonTriggered: function() {
+			$( '.give-save-button' ).on( 'click', function () {
```

Comment:
@mehul0810 This should be setting page specific. We can have `.give-save-button` at multiple place.

Reply:
@ravinderk Updated the code to be more specific for settings page.